tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<p>invidual -&gt; individual</p><p> </p><p>The <a href="https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb" target="_blank">style guide</a> encourages imperative style.</p><p> </p><p>E.g. -&gt; "Use the <code>cluster_weights</code>  API to cluster either the entire model or individual layers."</p><p> </p><p>Similarly it's recommended to not use "we" unless necessary. See the quantization one as an example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>